structured.js

```diff
@@ -585,7 +599,34 @@
                     right: {type: "Literal",
                             value: 1,
                             raw: "1"}};
-        }
+        } /*else if ("body" in currTree && !options.notvar) {
```

Comment:
Can you put a comment about why we chose to comment it out? (This would make bla and bla equivalent, but then while(true) { var x = 5; } var y = 6; will match var x = 5; var y = 6;, which we don't think we want.)
Alternatively, just delete the commented out code. We can retrieve it in the PR history if desired.